test/e2e/storage/drivers/csi.go

```diff
@@ -434,3 +445,30 @@ func (g *gcePDCSIDriver) PrepareTest(f *framework.Framework) (*testsuites.PerTes
 			cancelLogging()
 		}
 }
+
+const csiNodeRegisterTimeout = 1 * time.Minute
```

Comment:
nit: Either keep the const inside the function scope, or move it to the top of the file